src/runtime/runtime.cs

```diff
@@ -473,6 +473,19 @@ private static void ResetPyMembers()
         internal static IntPtr PyNone;
         internal static IntPtr Error;
 
+        public static PyObject _none;
+        public static PyObject GetPyNone()
+        {
+            if (_none == null)
+            {
+                var result = Runtime.PyNone;
+                Runtime.XIncref(result);
+                _none= new PyObject(result);
```

Comment:
I do not think there's much point in caching result.

But if you do, you must deal with:
1. When runtime is shutdown and restarted, this value should be reinitialized.
2. On the .NET side writing to `_none` must be properly synchronized. E.g. you need a lock or an atomic swap.

Again, however, I think you can just not cache the value.

Reply:
ok, I best remove the caching